CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+## 1.2.0
+
+### Minor Changes
+
+* Add support for overriding the EncryptionContext in DynamoDBEncryptor #60
```

Comment:
A Y version change is minor, a major change is expected to break backwards compatibility.

What do you think about what it would look like if we were to split it into minor changes and bugfixes, then remove the maintenance items?

Reply:
Sounds good.